packages/schematics/angular/ssr/schema.json

```diff
@@ -15,6 +15,12 @@
       "description": "Skip installing dependency packages.",
       "type": "boolean",
       "default": false
+    },
+    "serverRouting": {
+      "description": "Creates a server application using the Server Routing and App Engine APIs (Developer Preview).",
+      "x-prompt": "Would you like to use the Server Routing and App Engine APIs (Developer Preview) for this server application?",
```

Comment:
Once we have documentation for this, it might be helpful to link to it from here so users can understand what we're asking about and whether they're interested in it.